compiler/src/iree/compiler/Codegen/Dialect/IREECodegenAttrs.td

```diff
@@ -124,12 +126,18 @@ def IREECodegen_TranslationInfoAttr :
     OptionalParameter<"unsigned",
         "The software pipeline depth to be used">:$softwarePipelineDepth,
     DefaultValuedParameter<"unsigned", "1",
-        "The software pipeline stage to place stores">:$softwarePipelineStoreStage
+        "The software pipeline stage to place stores">:$softwarePipelineStoreStage,
+    OptionalParameter<"StringAttr",
```

Comment:
I am not sure adding a file name path to the translation info is a good idea... Could we just keep the symbol name here, and have the transform dialect file be provided in someother way. They seem to be decoupled things.

Reply:
The filename is nice to have but I can drop it in this PR. I'm not sure what the alternate place would be though because a transform script is inherently representative of the translation info for the dispatch. Can worry about it later.

Reply:
I'd be +1 re avoiding another CLI for new paths and instead using the existing if possible (or even better, retire the 2 existing flags that are target dependent and use a single flag).
In the first step we could just use the presence of the `SymbolRefAttr` to use a named sequence vs the top-level `transform.sequence`.
Then we can update/retire old tests to use this mechanism and wrap them in named sequnces.
Then we can require named sequences for everything and build the selection I mention in the top-level comment.